pandas/tests/tools/test_to_numeric.py

```diff
@@ -649,3 +649,10 @@ def test_failure_to_convert_uint64_string_to_NaN():
     ser = Series([32, 64, np.nan])
     result = to_numeric(pd.Series(["32", "64", "uint64"]), errors="coerce")
     tm.assert_series_equal(result, ser)
+
+
+def test_precision():
```

Comment:
can you parameterize this and add the other examples from the OP (currently working an non-working ones that are fixed here).

can you rename this to test_precision_float_conversion

Reply:
OK, I just copied the huge list of numbers that he had there and put it in the test.  It does fail in master, works with this patch.